### PR TITLE
CON-3823 Fixed menu creation when items exist

### PR DESCRIPTION
**Pull request by sbofirov**

### CODE REVIEW

**Comment by mkafadarov** [comment]
I am ok with this solution

File: plugin.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "license": "proprietary",
     "link": "http://store.shopware.com",
     "author": "Shopware AG",
-    "currentVersion": "1.0.2",
+    "currentVersion": "1.0.3",
```
Comment (svenfinke) [inline plugin.json]: Damn... I forgot to increase the version number. The new version will be 1.0.4. Can you increase it?

**Comment by sbofirov** [comment]
@svenfinke done

**Comment by svenfinke** [comment]
I'm also fine with it. I'll give this to the QA so that they can test it

**Pull request merged**